lib/r10k/api/git.rb

```diff
@@ -20,6 +20,14 @@ def rev_parse(rev, opts={})
         provider.rev_parse(rev, opts)
       end
 
+      def fetch(remote, opts={})
+        provider.fetch(remote, opts)
+      end
+
+      def clone(remote, opts={})
+        provider.clone(remote, opts)
+      end
```

Comment:
It seems like the signature for clone should have a target path argument and an option to control whether or not it's a bare clone. We could probably clean that up with RK-207 though.